### PR TITLE
(13.4.0) INFRASYS-7486 Gives users the option to use autoscaling groups from cloudformation stack

### PR DESCRIPTION
**Pull request by sbraverman**

### CODE REVIEW

File: License2Deploy/rolling_deploy.py [hunk 1]
```diff
@@ -11,19 +11,32 @@ class RollingDeploy(object):
...
+               ami_id=None,
+               profile_name=None,
+               regions_conf=None,
+               stack_id=None,
+               session=None):
     self.env = env
+    self.session = session
     self.project = project.replace('-','')
-    self.buildNum = buildNum
+    self.buildNum = build_number
```
Comment (sbraverman) [inline License2Deploy/rolling_deploy.py]: self.buildNum should be snake_case

File: README.md [hunk 2]
```diff
@@ -20,25 +20,27 @@ The rolling deployment will:
...
+  -a AMI_ID, --ami AMI_ID
                         AMI ID
   -P PROFILE, --profile PROFILE
                         Profile name as designated in aws credentials/config
                         files
   -c CONFIG, --config CONFIG
                         Config file Location, eg.
                         /opt/License2Deploy/config.yml
+  -s STACK_NAME, --stack STACK_NAME
+                        Sack name if AutoScaling Group created via CloudFormation
```
Comment (sbraverman) [inline README.md]: stack

**Comment by nz285** [comment]
+1

**Comment by mayn** [comment]
+1

File: License2Deploy/rolling_deploy.py [hunk 3]
```diff
@@ -296,12 +321,13 @@ def get_args(): # pragma: no cover
...
   parser.add_argument('-P', '--profile', default='default', action='store', dest='profile', help='Profile name as designated in aws credentials/config files', type=str)
   parser.add_argument('-c', '--config', default='/opt/License2Deploy/regions.yml', action='store', dest='config', help='Config file Location, eg. /opt/License2Deploy/regions.yml', type=str)
+  parser.add_argument('-s', '--stack', action='store', dest='stack_name', help='Stack name if AutoScaling Group created via CloudFormation', type=str)
   return parser.parse_args()
 
 def main(): # pragma: no cover
   args = get_args()
   SetLogging.setup_logging()
-  deployObj = RollingDeploy(args.env, args.project, args.buildNum, args.amiID, args.profile, args.config)
+  deployObj = RollingDeploy(args.env, args.project, args.buildNum, args.amiID, args.profile, args.config, args.stack_name)
```
Comment (taoistmath) [inline License2Deploy/rolling_deploy.py]: Did we want to change buildNum here (and in get_args) to build_number to be consistent?

**Comment by taoistmath** [comment]
One comment otherwise +1

**Pull request merged**